nobl9/resource_direct_test.go

```diff
@@ -323,6 +324,35 @@ resource "nobl9_direct_%s" "%s" {
 `, directType, name, name, testProject)
 }
 
+func testLogicMonitorDirect(directType, name string) string {
+	return fmt.Sprintf(`
+resource "nobl9_direct_%s" "%s" {
+  name = "%s"
+  project = "%s"
+  description = "desc"
+  account = "account_name"
+  account_id = "secret"
+  access_key = "secret"
+  log_collection_enabled = true
+  release_channel = "beta"
+  historical_data_retrieval {
```

Comment:
The `historical_data_retrieval` block should be removed since we don't have support for the query engine for now.